ee/packages/license/src/AirGappedRestriction.ts

```diff
@@ -0,0 +1,73 @@
+import EventEmitter from 'events';
+
+import { License } from '.';
+import { decryptStatsToken } from './token';
+
+const DAY_IN_MS = 24 * 60 * 60 * 1000;
+const NO_ACTION_PERIOD_IN_DAYS = 3;
+const WARNING_PERIOD_IN_DAYS = 7;
+
+class AirGappedRestrictionClass extends EventEmitter {
+	#restricted = false;
```

Comment:
it was before, I changed it after discussing with Tasso

Reply:
Any reason to follow this pattern?

Reply:
the `private` keyword is provided by TS, so this doesn't guarantee that the property is actually private since typechecking is usually done at compile time. There was also an issue with the naming convetion. See [tassoevan's comment above](https://github.com/RocketChat/Rocket.Chat/pull/33241#discussion_r1790521663).

But as in a Linkin Park song:
> In the end, it doesn't even matter

Reply:
I just went research it. For reference https://stackoverflow.com/questions/59641564/what-are-the-differences-between-the-private-keyword-and-private-fields-in-types